<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-11-23T20:31:03 IST" name="Assessment" finished-at="2020-11-23T20:31:16 IST" duration-ms="12883">
    <groups>
    </groups>
    <test started-at="2020-11-23T20:31:03 IST" name="Login and create account error message" finished-at="2020-11-23T20:31:16 IST" duration-ms="12883">
      <class name="assessmentpackage.TC_02Test">
        <test-method is-config="true" signature="setUp()[pri:0, instance:assessmentpackage.TC_02Test@3059cbc]" started-at="2020-11-23T20:30:16 IST" name="setUp" finished-at="2020-11-23T20:31:03 IST" duration-ms="47084" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validate_error_msg()[pri:0, instance:assessmentpackage.TC_02Test@3059cbc]" started-at="2020-11-23T20:31:03 IST" name="validate_error_msg" finished-at="2020-11-23T20:31:16 IST" duration-ms="12808" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message is displayed when email ID is invalid during sign in expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message is displayed when email ID is invalid during sign in expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at assessmentpackage.login_signup_bug.validate_error_message(login_signup_bug.java:82)
at assessmentpackage.TC_02Test.validate_error_msg(TC_02Test.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validate_error_msg -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:assessmentpackage.TC_02Test@3059cbc]" started-at="2020-11-23T20:31:17 IST" name="tearDown" finished-at="2020-11-23T20:31:18 IST" duration-ms="1195" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- assessmentpackage.TC_02Test -->
    </test> <!-- Login and create account error message -->
  </suite> <!-- Assessment -->
</testng-results>
